modules/pel/peltool/peltool.py

```diff
@@ -713,68 +713,94 @@ def main():
     PELsPath = "/var/lib/phosphor-logging/extensions/pels/logs/"
     PELsArchivePath = "/var/lib/phosphor-logging/extensions/pels/logs/archive"
     inBMC = os.path.isdir(PELsPath)
-    parser = argparse.ArgumentParser(formatter_class=CustomFormatter, description="PELTools")
+    example_text = '''Examples:
+        -------------------------
+        Display summary of informational PELs along with serviceable PELs: peltool.py -l -S Informational
```

Comment:
I think anytime we use a '-l', we should use 'list' in the discription.  I know the --plid, --src, etc, break that by doing a list without  the -l, but oh well.